instance/api/openedx_appserver.py

```diff
@@ -98,5 +98,5 @@ def make_active(self, request, pk):
             return Response(
                 {"error": "Cannot make an unhealthy app server active."}, status=status.HTTP_400_BAD_REQUEST
             )
-        app_server.instance.set_appserver_active(app_server.pk)
-        return Response({'status': 'App server updated.'})
+        set_appserver_active(app_server.pk)
+        return Response({'status': 'App server activation initiated.'})
```

Comment:
These fixes are already live, and the active AppServer should be updated correctly.  I've got no idea why you are still seeing "App server updated", since this string doesn't occur anywhere in the currently running code (I used grep on console.opencraft.com to double check).

The traceback you included is happening while installing the requirements, but the installation succeeds anyway.  I haven't investigated what exactly is causing the traceback, but since pip returns with a success exit code I presume that the error is immaterial.

The log is attached to the load balancer, and not to a particular instance.  Since load balancers aren't shown in the UI yet, this means the messages aren't visible in the UI for now.  It doesn't make sense to attach the messages to an instance, since the reconfiguration of the load balancer happens for all attached instances at once, so there isn't a single instance to attach it to.